login-workflow/src/screens/ForgotPasswordScreen/ForgotPasswordScreenBase.tsx

```diff
@@ -55,9 +55,11 @@ export const ForgotPasswordScreenBase: React.FC<React.PropsWithChildren<ForgotPa
     const instructionsProps = props.WorkflowCardInstructionProps || {};
     const actionsProps = props.WorkflowCardActionsProps || {};
 
-    const [isEmailValid, setIsEmailValid] = useState(emailValidator(initialEmailValue) ?? false);
-    const [emailError, setEmailError] = useState('');
-    const [shouldValidateEmail, setShouldValidateEmail] = useState(false);
+    const validateEmail = (): boolean => typeof emailValidator(initialEmailValue) === 'string' ? false : true;
```

Comment:
We can minimize this to:
    const validateEmail = (): boolean => typeof emailValidator(initialEmailValue) !== 'string';

Reply:
Done. Please review.